terracotta/scripts/connect.py

```diff
@@ -20,6 +20,18 @@
 from terracotta.scripts.http_utils import find_open_port
 
 
+def build_request(url: str) -> urllib.request.Request:
+    """
+    Build a request object with headers that mimic a web browser.
```

Comment:
Could you please add a comment as to *why* we do this?

Reply:
Done!